src/contractMetadata.ts

```diff
@@ -1,5 +1,5 @@
 
-import * as contractData from './contractMetadata.json';
+import contractData from './contractMetadataData';
```

Comment:
Along with the comment/change above for not using default exports, this should be changed to something like:
```suggestion
import { contractMetadata } from './contractMetadataData';
```

Reply:
Same comment elsewhere.